docs/src/man/hosting/walkthrough.md

```diff
@@ -135,6 +135,7 @@ read [Generating an SSH Key](@ref).
 First, we need to Base64 encode the private key. Open Julia, and run the command
 
 ```julia
+julia> using Base64
```

Comment:
```suggestion
julia> using Base64

```

Reply:
Is this just a new line after the first command?

Reply:
Yep. This brings it in line with how it would actually be displayed in the REPL.